samples/snippets/detect_intent_stream_test.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020, Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for detect_intent_texts."""
+
+from __future__ import absolute_import
+
+import os
+import uuid
+
+
+from detect_intent_stream import detect_intent_stream
+
+DIRNAME = os.path.realpath(os.path.dirname(__file__))
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+AGENT_ID = "53516802-3e2a-4016-80b6-a3df0d240240"
+AGENT = "projects/{}/locations/global/agents/{}".format(PROJECT_ID, AGENT_ID)
+SESSION_ID = uuid.uuid4()
+AUDIO = "{}/resources/hello.wav".format(DIRNAME)
```

Comment:
The location for external resources for the test should be passed in by env vars - See [External Resources](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#external-resources)